main.go

```diff
@@ -86,8 +87,7 @@ func main() {
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
```

Comment:
where do we define the port now?

Reply:
from what I have seen its seems to be bundled with metricAddr I wish if they called `BindSocket` instead

Reply:
I need to investigate further this port is used by webhook server 
```
	// Port is the port that the webhook server serves at.
	// It is used to set webhook.Server.Port if WebhookServer is not set.
	//
	// Deprecated: Use WebhookServer instead. A WebhookServer can be created via webhook.NewServer.
	Port int
```

Reply:
fixed